### PR TITLE
EREGCSC-1577 BASE_URL in app; removing hard-coded prod domain name

### PR DESCRIPTION
**Pull request by PhilR8**
Resolves [EREGCSC-1577](https://jiraent.cms.gov/browse/EREGCSC-1577)

**Description**
The Resources test suite for Cypress was failing only when being run against production due to the AWS URL having a subdirectory (`random-guid.amazonaws.com/prod`) and the custom domain name not having one (`regulations-pilot.cms.gov`).  This was due to how we were setting the `base` property of the Vue router.  This has been fixed.

**This pull request changes**

- Pulls `CUSTOM_URL` from AWS Parameter Store to be used as canonical custom domain name
- Compares `CUSTOM_URL` with `host` (provided by Django) to properly set Vue Router logic on prod

**Steps to manually verify this change**

2. Deploy to prod and check that Resources test suite passes and the site works as expected

### CODE REVIEW

File: solution/ui/regulations/eregs-vite/src/utilities/api.js [hunk 1]
```diff
@@ -821,7 +820,8 @@ const getSupByPart = async (title, part, subparts, sections) => {
 }
 const getCategories = async () =>  httpApiGetV3("resources/categories");
 
-const getTOC = async (title) =>  httpApiGetV3(title? `title/${title}/toc`:`toc`);
+const getTOC = async (title) =>
+    httpApiGetV3(title ? `title/${title}/toc` : `toc`);
```
Comment (PhilR8) [inline solution/ui/regulations/eregs-vite/src/utilities/api.js:825]: prettier formatting

File: solution/ui/regulations/eregs-vite/src/utilities/api.js [hunk 2]
```diff
@@ -217,8 +222,8 @@ function httpApiGetV3(urlPath, { params } = {}) {
 }
 
 async function httpApiGetV3WithPagination(urlPath, { params } = {}) {
-    let results = []
-    let url = `${config.apiPathV3}/${urlPath}`
+    let results = [];
+    let url = `${config.apiPathV3}/${urlPath}`;
```
Comment (PhilR8) [inline solution/ui/regulations/eregs-vite/src/utilities/api.js:226]: prettier formatting

File: solution/ui/regulations/eregs-vite/src/utilities/api.js [hunk 3]
```diff
@@ -821,7 +820,8 @@ const getSupByPart = async (title, part, subparts, sections) => {
 }
 const getCategories = async () =>  httpApiGetV3("resources/categories");
 
-const getTOC = async (title) =>  httpApiGetV3(title? `title/${title}/toc`:`toc`);
+const getTOC = async (title) =>
+    httpApiGetV3(title ? `title/${title}/toc` : `toc`);
```
Comment (PhilR8) [inline solution/ui/regulations/eregs-vite/src/utilities/api.js:824]: prettier formatting

**Review by peggles2 — APPROVED** [review]
LGTM 👍

**Pull request merged**